public/app/features/alerting/unified/mocks/server/handlers/mimirRuler.ts

```diff
@@ -0,0 +1,74 @@
+import { delay, http, HttpResponse } from 'msw';
+
+import { RulerRuleGroupDTO } from '../../../../../../types/unified-alerting-dto';
+import { namespaces } from '../../mimirRulerApi';
+import { HandlerOptions } from '../configure';
+import { MIMIR_DATASOURCE_UID } from '../constnts';
+
+export const updateRulerRuleNamespaceHandler = (options?: HandlerOptions) => {
```

Comment:
nit: I've been aiming to only export the handler methods when they're needed in the `configure` methods (but this is a nit because we have unnecessary exports elsewhere so its not critical 😁 )